server_addon/maya/client/ayon_maya/plugins/publish/collect_fbx_model.py

```diff
@@ -0,0 +1,29 @@
+import pyblish.api
+from ayon_core.pipeline import OptionalPyblishPluginMixin
+from ayon_maya.api import plugin
+
+
+
+class CollectFbxModel(plugin.MayaInstancePlugin,
+                      OptionalPyblishPluginMixin):
+    """Collect Camera for FBX export."""
+
+    order = pyblish.api.CollectorOrder + 0.2
+    label = "Collect Model for FBX export"
```

Comment:
This would be quite an odd label in publisher UI for the checkbox to toggle the FBX export on/off for a model family? If this were just "Export FBX (model)" or alike then it'd be I think a better checkbox in the publisher UI but maybe an odd label for the Collector itself.

---

Also, this highlights again the issues with how the publishing logic works that it requires this odd family "ingesting" from a collector to support the FBX export. Still feels odd we need to do it this way but can't think of anything easier within current design. "Product aliases" and other things have been mentioned in the past and also a lot of "yes yes - should be improved" has come up often. Would still be very interested in seeing proposals in this area or having momentum of pushing that forward. The longer we are where we are the bigger this codebase will just grow with these workarounds that feel out-of-place yet unfortunately are required for now.